### PR TITLE
[WIP] TF-3335 Improve move email jmap requests

### PR DESCRIPTION
**Pull request by tddang-linagora**
## Issue
- #3335 

## Demo
Updating

### CODE REVIEW

File: lib/features/mailbox/presentation/mailbox_controller.dart [hunk 1]
```diff
@@ -272,10 +270,6 @@ class MailboxController extends BaseMailboxController
           _refreshMailboxChanges(currentMailboxState: success.currentMailboxState);
         } else if (success is MarkAsMultipleEmailReadHasSomeEmailFailure) {
           _refreshMailboxChanges(currentMailboxState: success.currentMailboxState);
```
Comment (chibenwa) [inline lib/features/mailbox/presentation/mailbox_controller.dart:272]: Whay not handle flags update at the same time?

**Comment by tddang-linagora** [comment]
Already covered in PR
- https://github.com/linagora/tmail-flutter/pull/3362

**Comment by tddang-linagora** [comment]
**Partially** covered in PR
- #3334

**Comment by tddang-linagora** [comment]
>Partially covered in PR

Final edit. It **DID** completely cover.

**Pull request closed without merging**